docs/gallery/tiled-random-samples.ipynb

```diff
@@ -0,0 +1,155 @@
+    "    np.random.logseries: {'p': [0.25, 0.5, 0.75]},\n",
+    "    np.random.pareto: {'a': [16, 24, 32]},\n",
+    "    np.random.geometric: {'p': [0.5, 0.7, 0.9]},\n",
+    "    np.random.power: {'a': [1, 2, 4]},\n",
+    "    np.random.triangular: {'left': [0], 'right': [1], 'mode': [0.2, 0.4, 0.8]},\n",
+    "    np.random.vonmises: {'mu': [0], 'kappa': [1, 2, 4]},\n",
+    "    np.random.exponential: {'scale': [1, 2, 3]},\n",
+    "    np.random.wald: {'mean': [1], 'scale': [16, 32, 64]},\n",
+    "    np.random.weibull: {'a': [2, 4, 8, 16]},\n",
+    "    np.random.f: {'dfnum': [100], 'dfden': [100]},\n",
+    "    np.random.rayleigh: {'scale': [2, 3, 4]}\n",
+    "}\n",
+    "\n",
+    "def generate_kwargs(arguments_map: dict[str, list]) -> Generator[dict, None, None]:\n",
+    "    from itertools import product\n",
+    "\n",
+    "    key_arg_pairs = [[(key, val) for val in values] for key, values in arguments_map.items()]\n",
+    "    for arguments in product(*key_arg_pairs):\n",
+    "        yield {str(args[0]): args[1] for args in arguments}\n",
+    "\n",
+    "def key_arg_name(key: str, arg) -> str:\n",
+    "    return \": \".join([key, str(arg)])\n",
+    "\n",
+    "def keys_args_name(**kwargs) -> str:\n",
+    "    return \",\".join([key_arg_name(key, arg) for key, arg in kwargs.items()])\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Collect samples\n",
+    "def sample_and_hist(pdf: Callable, size: int, **kwargs):\n",
+    "    samples = sc.array(dims=['event'], values=pdf(size=size, **kwargs), unit='1')\n",
+    "    \n",
+    "    return samples.hist(event=10)/len(samples)\n",
+    "\n",
+    "plots = dict()\n",
+    "\n",
+    "for pdf, arguments_map in pdf_maps.items():\n",
+    "    hists = {\n",
+    "        keys_args_name(**kwargs): sample_and_hist(pdf, size=10_000, **kwargs)\n",
+    "        for kwargs in generate_kwargs(arguments_map)    \n",
+    "    }\n",
+    "    plot = pp.plot(hists, title=f\"{pdf.__name__} distribution\", alpha=0.8, linewidth=4)\n",
+    "    plots[pdf.__name__] = plot\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Show all samples\n",
+    "tiled = pp.tiled(5, 4)\n",
+    "plot_table = np.array(list(plots.values())).reshape((5, 4))\n",
+    "for i_row in range(5):\n",
+    "    for i_col in range(4):\n",
+    "        tiled[i_row, i_col] = plot_table[i_row][i_col]\n",
+    "        tiled[i_row, i_col].ax.legend(fontsize=8)\n",
+    "\n",
+    "tiled"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Continuous samples\n",
+    "continuous_samples = plots['normal'] + plots['laplace'] + plots['vonmises']\n",
+    "continuous_samples.fig.set_size_inches((15, 3))\n",
```

Comment:
I wonder if this (and the one below) is needed for the docs? I think the figure might get scaled to the page width automatically? You could then reduce the amount of code by just having `plots['binomial'] + plots['logseries']` in the cell?

Reply:
That is true... it was only annoying in my notebook....! I'll remove them.